apps/dashboard/pkg/migration/schemaversion/v34.go

```diff
@@ -0,0 +1,294 @@
+//
+// This migration addresses CloudWatch queries where a single query uses multiple statistics
+// (e.g., statistics: ['Max', 'Min']). The migration splits these into separate queries,
+// each with a single statistic (e.g., one query with statistic: 'Max', another with statistic: 'Min').
+//
+// The migration works by:
+// 1. Identifying CloudWatch queries in panel targets that have a 'statistics' array
+// 2. Creating separate queries for each statistic in the array
+// 3. Replacing the original 'statistics' array with a single 'statistic' field
+// 4. Generating new refIds for additional queries (B, C, D, etc.)
+// 5. Applying the same logic to CloudWatch annotation queries
+// 6. Adding statistic suffixes to annotation names when multiple annotations are created
+//
+// Panel Query Example - Multiple Statistics:
+//
+// Before migration:
+//
+//	target: {
+//	  refId: "A",
+//	  dimensions: {"InstanceId": "i-123"},
+//	  namespace: "AWS/EC2",
+//	  region: "us-east-1",
+//	  metricName: "CPUUtilization",
+//	  statistics: ["Average", "Maximum", "Minimum"]
+//	}
+//
+// After migration:
+//
+//	targets: [
+//	  { refId: "A", dimensions: {"InstanceId": "i-123"}, namespace: "AWS/EC2", region: "us-east-1", metricName: "CPUUtilization", statistic: "Average" },
+//	  { refId: "B", dimensions: {"InstanceId": "i-123"}, namespace: "AWS/EC2", region: "us-east-1", metricName: "CPUUtilization", statistic: "Maximum" },
+//	  { refId: "C", dimensions: {"InstanceId": "i-123"}, namespace: "AWS/EC2", region: "us-east-1", metricName: "CPUUtilization", statistic: "Minimum" }
+//	]
+//
+// Annotation Query Example - Multiple Statistics:
+// Before migration:
+//
+//	annotation: {
+//	  name: "CloudWatch Alerts",
+//	  dimensions: {"InstanceId": "i-123"},
+//	  namespace: "AWS/EC2",
+//	  region: "us-east-1",
+//	  prefixMatching: false,
+//	  statistics: ["Maximum", "Minimum"]
+//	}
+//
+// After migration:
+//
+//	annotations: [
+//	  { name: "CloudWatch Alerts - Maximum", dimensions: {"InstanceId": "i-123"}, namespace: "AWS/EC2", region: "us-east-1", prefixMatching: false, statistic: "Maximum" },
+//	  { name: "CloudWatch Alerts - Minimum", dimensions: {"InstanceId": "i-123"}, namespace: "AWS/EC2", region: "us-east-1", prefixMatching: false, statistic: "Minimum" }
+//	]
+func V34(dashboard map[string]interface{}) error {
+	dashboard["schemaVersion"] = int(34)
+
+	// Migrate panel queries if panels exist
+	panels, ok := dashboard["panels"].([]interface{})
+	if ok {
+		for _, panel := range panels {
+			p, ok := panel.(map[string]interface{})
+			if !ok {
+				continue
+			}
+
+			migrateCloudWatchQueriesInPanel(p)
+
+			// Handle nested panels in collapsed rows
+			if nestedPanels, hasNested := p["panels"].([]interface{}); hasNested {
+				for _, nestedPanel := range nestedPanels {
+					if np, ok := nestedPanel.(map[string]interface{}); ok {
+						migrateCloudWatchQueriesInPanel(np)
+					}
+				}
+			}
```

Comment:
nit (so feel free to ignore), but slight preference towards less nesting
```suggestion
			nestedPanels, hasNested := p["panels"].([]interface{})
			if !hasNested {
				continue
			}
			for _, nestedPanel := range nestedPanels {
				if np, ok := nestedPanel.(map[string]interface{}); ok {
					migrateCloudWatchQueriesInPanel(np)
				}
			}
```